Makefile

```diff
@@ -210,10 +214,18 @@ ifneq ($(USE_GPERFTOOLS), 1)
 				endif
 			endif
 		endif
+		JEMALLOC_CUSTOM=$(shell pkg-config --exists jemalloc; echo $$?)
+		ifeq ($(JEMALLOC_CUSTOM), 0)
+			CFLAGS += $(shell pkg-config --cflags jemalloc)
+			LDFLAGS += $(shell pkg-config --libs jemalloc)
```

Comment:
Maybe use a name that's more informative, such as USE_JEMALLOC_PATH, and let the none case perform the pkg-config look-up? BTW thanks for making this change. This is useful for statically linking jemalloc.

Reply:
Are you suggesting to make this a CLI option that one can pass to `make` ?

Reply:
I was suggesting that we use this in [build config](https://github.com/apache/incubator-mxnet/blob/master/make/config.mk).